### ISSUE TITLE
Support for first party Combine framework

### ISSUE THREAD

**Issue by edorphy** [kinetic-fit/sensors-swift#7]
A great enhancement would be to refresh this library with Combine.

**Comment by netizen01** [comment]
Absolutely! Need to maintain backwards compatibility for a while, but we'll get there.

**Comment by netizen01** [comment]
Following up:

**Combine Availability:**
iOS 13.0+
macOS 10.15+
Mac Catalyst 13.0+
tvOS 13.0+
watchOS 6.0+

I will look into a v2 of this library this fall with Combine instead of Signals with the release of iOS 14 and macOS 11.

**Comment by edorphy** [comment]
Any updates? I'm writing my own lib and putting @Published properties all over the place and it works out nicely.

**Comment by netizen01** [comment]
Still keeping an eye on this. Now that iOS15 is out I will start a V2 branch where we can work on Combine support.
